lib/src/locator.dart

```diff
@@ -1,32 +1,46 @@
 import 'package:get_it/get_it.dart';
+import 'package:isar/isar.dart';
 import 'package:kana_to_kanji/src/core/repositories/settings_repository.dart';
-import 'package:kana_to_kanji/src/core/services/database_service.dart';
 import 'package:kana_to_kanji/src/core/models/group.dart';
 import 'package:kana_to_kanji/src/core/models/kana.dart';
 import 'package:kana_to_kanji/src/core/repositories/groups_repository.dart';
 import 'package:kana_to_kanji/src/core/repositories/kana_repository.dart';
+import 'package:kana_to_kanji/src/core/services/api_service.dart';
+import 'package:kana_to_kanji/src/core/services/groups_service.dart';
 import 'package:kana_to_kanji/src/core/services/info_service.dart';
+import 'package:kana_to_kanji/src/core/services/kana_service.dart';
 import 'package:kana_to_kanji/src/core/services/preferences_service.dart';
 import 'package:logger/logger.dart';
 
 final GetIt locator = GetIt.instance;
 
 void setupLocator() {
   locator.registerLazySingleton<Logger>(() => Logger());
-  locator.registerSingletonAsync<DatabaseService>(() async {
-    final instance = DatabaseService();
 
-    await instance.initialize([Group.tableCreate, Kana.tableCreate], []);
-
-    return instance;
-  });
+  // Services
+  locator.registerLazySingleton<GroupsService>(() => GroupsService());
+  locator.registerLazySingleton<KanaService>(() => KanaService());
```

Comment:
In the end, because they are only used and needed by their respective repository, I don't see the reason to have them registered in the locator 🤔 

**EDIT**: They are used for `loadCollection` so never mind me

Reply:
Retiré vu qu'il n'y a plus besoin de les avoir dans les appeler dans le main